test/unit/warp-routes.test.ts

```diff
@@ -67,5 +67,29 @@ describe('Warp Route Configs', () => {
       // Verify the chain names match
       expect(chainNames).to.deep.equal(derivedChainNames, 'Chain names in ID must match derived chain names');
     });
+
+    it(`Route ${id} only specifies a coinGeckoId for tokens that escrow tokens`, () => {
+      const config = routes[id];
+
+      const warpCore = WarpCore.FromConfig(multiProvider, config);
+      
+      for (const token of warpCore.tokens) {
+        if (token.coinGeckoId === undefined) {
+          continue;
+        }
+        // At the moment, the SDK doesn't have a super clean way
```

Comment:
Mind leaving a TODO comment too, like:
`// TODO add TOKEN_SYNTHETIC_STANDARDS list to SDK`